iothub_client/tests/iothub_client_authorization_ut/iothub_client_authorization_ut.c

```diff
@@ -387,6 +401,11 @@ TEST_FUNCTION(IoTHubClient_Auth_Create_fail)
     size_t count = umock_c_negative_tests_call_count();
     for (size_t index = 0; index < count; index++)
     {
+        if (index == 1)
+        {
+            continue;
+        }
```

Comment:
are you skipping the test of the base64 decode here?

Reply:
No, `Azure_Base64_Decode` is the `0`. We are skipping `BUFFER_delete`.

---
In reply to: [334060966](https://github.com/Azure/azure-iot-sdk-c/pull/1250#discussion_r334060966) [](ancestors = 334060966)